base/dict.jl

```diff
@@ -686,6 +686,115 @@ end
 
 filter!(f, d::Dict) = filter_in_one_pass!(f, d)
 
+
+
+
+
+
+
+"""
+    map!(f, values(dict))
+Takes the function f(value) and transforms values stored in the dict with the transformation value=f(value).
```

Comment:
one blank line after `map!` line, and only one blank line before `# Examples` 

Also quote code `f(value)`, `dict`, and `value=f(value)`